docs/how-to-guides/blob-migration.md

```diff
@@ -0,0 +1,50 @@
+# Blob migration
+
+Currently DICOM files are stored with DICOM UIDs as blob names in blob storage, using the template `{account}/{container}/{studyUid}/{seriesUid}/{sopInstanceUid}_{watermark}.dcm`. 
+Since UIDs may include personal information about the context of their creation, such as patient information or identifiers. So we made the decision to change the way that we store DICOM files. In the next sections we have detailed steps for each scenario to migrate your existing blobs from old format to new format.
+
+## Blob migration configuration
+Below is the appsettings configuration for blob migration. Few properties needs to be updated to trigger migration.
+
+```json
+"DicomServer": {
+    "Services": {
+      "BlobMigration": {
+        "FormatType": "Old",
+        "StartCopy": false,
+        "StartDelete": false,
+        "CopyFileOperationId": "1d4689da-ca3b-4659-b0c7-7bf6c9ff25e1",
+        "DeleteFileOperationId": "ce38a27e-b194-4645-b47a-fe91c38c330f",
+        "CleanupDeletedFileOperationId": "d32a0469-9c27-4df3-a1e8-12f7f8fecbc8",
+        "CleanupFilterTimeStamp": "2022-08-01"
+      }
+}
+```
+
+## Migration steps
+
+1. If you are a new service and have not created any files, you can upgrade to the latest version of the service and skip the migration steps. That means you will see the BlobMigration.FormatType as "New" in appsettings configuration file. 
+
+2. If you have already uploaded DICOM files but does not care about previous data. You can use the [Delete API](../resources/conformance-statement.md#delete) to delete all the data which can free up space and then upgrade to the latest version of the service or you can upgrade to the latest version of the service and then use the [Delete API](../resources/conformance-statement.md#delete) to delete all the data.
+
+3. If you have already uploaded DICOM files and want to keep the previous data, you need to execute the below steps to copy the files first. This scenario has two options depending on whether if you want interruption to the service or not. Make sure Azure monitor is configured to monitor the service before starting the migration, for more info on how to configure Azure monitor, please refer to [Azure Monitor](../how-to-guides/configure-dicom-server-settings.md#azure-monitor).
+
+    3.1. If you are ok with interruption to the service, you can follow the steps below:
+
+        3.1.1. Change the BlobMigration.StartCopy to "True" and restart the service. This will start the copy background service which will trigger the CopyFiles Azure Durable Function which will copy the old format DICOM files to new format.
```

Comment:
We should add something in section 3 about Durable Functions setup or link to existing documentation.

```suggestion
        3.1.1. Set `BlobMigration.StartCopy` to `true` and restart the service. This will trigger the `CopyFiles` Durable Function which will copy the old format DICOM files to the new format.
```

Reply:
I was planning but there is no setup. If they deploy through our ARM template, it should be deployed already. Only thing they need to know is about Azure Monitor to look out for logs.